src/inausoft.netCLI.Tests/RootCommandHandlerTests.cs

```diff
@@ -11,30 +11,27 @@ public void RootCommandHandler_RunsProperCommandHandler_ForMultipleOptions()
         {
             //Arrange
             var stringOptionValue = "sampleString";
-            var pathOptionValue = @"C:\Program%Files";
             var intOptionValue = 102;
 
             var args = new string[] 
                             { 
                                 "command1",
                                 "--boolOption",
                                 "--stringOption", stringOptionValue,
-                                "--pathOption", pathOptionValue,
                                 "--intOption", intOptionValue.ToString()
                             };
 
             var mockCommandHandler = new MockCommand1Handler();
-            var rootCommandHanlder = new RootCommandHandler(new ICommandHandler[] { mockCommandHandler });
+            var config = new CLIConfiguration().Map<Command1, MockCommand1Handler>();
 
             //Act
-            var result = rootCommandHanlder.Run(args);
+            var result = netCLI.RunCLI(config, args, mockCommandHandler);
 
             //Assert
             Assert.AreEqual(0, result);
```

Comment:
I would add message that result other than 0 means errorcode meaning some errors

Reply:
Please elaborate.

Reply:
`Assert.AreEaqual(0, result, "RunCLI method indicted error in returned exit code "` 
I would like to explicit stress out why do we expect `0` here - returns of this method should be treated as exit codes

Reply:
Oh.. sure. Agreed.